app/templates/editar-partidas.html

```diff
@@ -57,63 +57,63 @@ <h3 class="side-bar-register-title">CADASTRO</h3>
+                                        <option value="masculino">Masculino</option>
+                                        <option value="feminino">Feminino</option>
+                                    </select>
+                                    <select id="time-a" name="time-a">
+                                        <option selected disabled>Selecione o time</option>
+                                        <option value="1-irc">1° IRC</option>
+                                        <option value="2-irc">2° IRC</option>
+                                        <option value="3-irc">3° IRC</option>
+                                        <option value="1-iaut">1° IAUT</option>
+                                        <option value="2-iaut">2° IAUT</option>
+                                        <option value="3-iaut">3° IAUT</option>
+                                        <option value="1-iedf">1° IEDF</option>
+                                        <option value="2-iedf">2° IEDF</option>
+                                        <option value="3-iedf">3° IEDF</option>
+                                        <option value="1-ieltm">1° IELTM</option>
+                                        <option value="2-ieltm">2° IELTM</option>
+                                        <option value="3-ieltm">3° IELTM</option>
+                                    </select>
+                                    <select id="times-b" name="times-b">
+                                        <option selected disabled>Selecione o time</option>
+                                        <option value="1-irc">1° IRC</option>
+                                        <option value="2-irc">2° IRC</option>
+                                        <option value="3-irc">3° IRC</option>
+                                        <option value="1-iaut">1° IAUT</option>
+                                        <option value="2-iaut">2° IAUT</option>
+                                        <option value="3-iaut">3° IAUT</option>
+                                        <option value="1-iedf">1° IEDF</option>
+                                        <option value="2-iedf">2° IEDF</option>
+                                        <option value="3-iedf">3° IEDF</option>
+                                        <option value="1-ieltm">1° IELTM</option>
+                                        <option value="2-ieltm">2° IELTM</option>
+                                        <option value="3-ieltm">3° IELTM</option>
+                                    </select>
+                                    <input class="input-data" type="datetime-local" name="date-game" id="date-game">
+                                </div>
+                                <div>
+                                    <input class="green-button" type="submit" value="Confirmar">
+                                </div>
```

Comment:
This table below has to be removed in the future, no things pre-ready in main

Essa tabela abaixo deverá ser retirada futuramente, nada de coisas pré-prontas na main